src/main/resources/native/ios/AppDelegate.m

```diff
@@ -74,7 +71,23 @@ @implementation AppDelegate
 
 -(void)startVM:(id)selector {
     gvmlog(@"Starting vm...");
-    startGVM();
+    NSArray *documentPaths = NSSearchPathForDirectoriesInDomains(NSLibraryDirectory, NSUserDomainMask, YES);
+    NSString *documentsDir = [documentPaths objectAtIndex:0];
+    NSString *folder = @"gluon";
+    NSString *folderPath = [documentsDir stringByAppendingPathComponent:folder];
+    NSString *prop = @"-Duser.home=";
+    NSString *userhomeProp = [prop stringByAppendingString: folderPath];
+
+    if (!folderPath) {
```

Comment:
Is this a critical error? Anyway, the check should be done before `NSString *userhomeProp = ...`?